### ISSUE TITLE
connect-salesforce all connectors 1.7.16 are failing

### ISSUE THREAD

**Issue by vdesabou** [vdesabou/kafka-docker-playground#227]
```
[2021-03-15 14:54:53,571] ERROR WorkerSourceTask{id=salesforce-pushtopic-source-0} Task threw an uncaught and unrecoverable exception. Task is being killed and will not recover until manually restarted (org.apache.kafka.connect.runtime.WorkerTask)
java.lang.LinkageError: loader constraint violation: when resolving method 'void org.eclipse.jetty.io.SelectorManager.<init>(java.util.concurrent.Executor, org.eclipse.jetty.util.thread.Scheduler, int)' the class loader org.apache.kafka.connect.runtime.isolation.PluginClassLoader @3da30852 of the current class, org/eclipse/jetty/client/AbstractConnectorHttpClientTransport$ClientSelectorManager, and the class loader 'app' for the method's defining class, org/eclipse/jetty/io/SelectorManager, have different Class objects for the type org/eclipse/jetty/util/thread/Scheduler used in the signature (org.eclipse.jetty.client.AbstractConnectorHttpClientTransport$ClientSelectorManager is in unnamed module of loader org.apache.kafka.connect.runtime.isolation.PluginClassLoader @3da30852, parent loader org.apache.kafka.connect.runtime.isolation.DelegatingClassLoader @7b94089b; org.eclipse.jetty.io.SelectorManager is in unnamed module of loader 'app')
        at org.eclipse.jetty.client.AbstractConnectorHttpClientTransport$ClientSelectorManager.<init>(AbstractConnectorHttpClientTransport.java:162)
        at org.eclipse.jetty.client.AbstractConnectorHttpClientTransport.newSelectorManager(AbstractConnectorHttpClientTransport.java:148)
        at org.eclipse.jetty.client.AbstractConnectorHttpClientTransport.doStart(AbstractConnectorHttpClientTransport.java:61)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
        at org.eclipse.jetty.client.HttpClient.doStart(HttpClient.java:255)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
        at io.confluent.salesforce.common.AbstractSalesforceSourceTask.createHttpClient(AbstractSalesforceSourceTask.java:501)
        at io.confluent.salesforce.common.AbstractSalesforceSourceTask.createClient(AbstractSalesforceSourceTask.java:511)
        at io.confluent.salesforce.pushtopic.SalesforcePushTopicSourceTask.createClient(SalesforcePushTopicSourceTask.java:79)
        at io.confluent.salesforce.common.AbstractSalesforceSourceTask.authenticateAndHandshake(AbstractSalesforceSourceTask.java:646)
        at io.confluent.salesforce.common.AbstractSalesforceSourceTask.start(AbstractSalesforceSourceTask.java:713)
        at org.apache.kafka.connect.runtime.WorkerSourceTask.execute(WorkerSourceTask.java:232)
        at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:185)
        at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:234)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by vdesabou** [comment]
Internal issue: CCDB-3790

**Comment by vdesabou** [comment]
Fixed with 1.7.17
